MAUI/Cards/getting-started.md

```diff
@@ -0,0 +1,202 @@
+---
+layout: post
+title: Getting started with MAUI Card control | Syncfusion
+description: Learn here all about getting started with Syncfusion .NET MAUI Card (SfCalendar) control and its basic features.
+platform: maui
+control: SfCard
+documentation: ug
+---
+
+# Getting started with .NET MAUI Card (SfCard)
+This section details the process of integrating the Card control and focuses solely on the fundamental features required for initiating your exploration of Syncfusion Card.
+
+To get start quickly with our .NET MAUI Card, you can check the below video.
+
+{% youtube
+"youtube:https://www.youtube.com/watch?v=kfQjKiD9Xas"%}
```

Comment:
Remove this- U added Calendar control video

Reply:
Removed the unwanted link